docs/user-guide/crawl-workflows.md

```diff
@@ -1,54 +1,37 @@
-# Crawl Workflows
+# Intro to crawl workflows
 
-Crawl Workflows consist of a list of configuration options that instruct the crawler what it should capture.
+Crawl workflows enable you to run crawls, which in turn produces an [archived item](./archived-items.md). A crawl workflow consists of a list of configuration options that instruct the crawler what it should capture and metadata about the workflow itself.
 
-## Creating and Editing Crawl Workflows
+## Create a Crawl Workflow
 
-New Crawl Workflows can be created from the Crawling page. A detailed breakdown of available settings can be found [here](workflow-setup.md).
+You can create new crawl workflows from the **Crawling** page, or the  _Create New ..._ shortcut from **Overview**.
 
-## Status
-
-Crawl Workflows inherit the [status of the last item they created](archived-items.md#status). When a workflow has been instructed to run it can have have five possible states:
-
-| Status | Description |
-| ---- | ---- |
-| <span class="status-waiting">:bootstrap-hourglass-split: Waiting</span>     | The workflow can't start running yet but it is queued to run when resources are available. |
-| <span class="status-waiting">:btrix-status-dot: Starting</span>       | New resources are starting up. Crawling should begin shortly.|
-| <span class="status-success">:btrix-status-dot: Running</span>        | The crawler is finding and capturing pages! |
-| <span class="status-waiting">:btrix-status-dot: Stopping</span> | A user has instructed this workflow to stop. Finishing capture of the current pages.|
-| <span class="status-waiting">:btrix-status-dot: Finishing Crawl</span> | The workflow has finished crawling and data is being packaged into WACZ files.|
-| <span class="status-waiting">:btrix-status-dot: Uploading WACZ</span> | WACZ files have been created and are being transferred to storage.|
-
-## Running Crawl Workflows
+### Choose what to crawl
 
-Crawl workflows can be run from the actions menu of the workflow in the crawl workflow list, or by clicking the _Run Crawl_ button on the workflow's details page.
+The first step in creating a new crawl workflow is to choose what you'd like to crawl. This determines whether the crawl type will be **URL List** or **Seeded Crawl**. Crawl types can't be changed after the workflow is created—you'll need to create a new crawl workflow.
 
-While crawling, the Watch Crawl page displays a list of queued URLs that will be visited, and streams the current state of the browser windows as they visit pages from the queue.
+#### Known URLs `URL List`{ .badge-blue }
 
-Running a crawl workflow that has successfully run previously can be useful to capture content as it changes over time, or to run with an updated [Crawl Scope](workflow-setup.md#scope).
+Crawl a single page, or choose this option if you already know the URL of every page you'd like to crawl. The crawler will visit every URL specified in a list, and optionally every URL linked on those pages.
```

Comment:
Not sure the "single page" bit is necessary? It's not distinct from knowing the url of every page you want to crawl. That said, it might still be worth including as a note?

```suggestion
Choose this option if you already know the URL of every page you'd like to crawl. The crawler will visit every URL specified in a list, and optionally every URL linked on those pages.
```